content/help/ops/security/debugging-authorization/index.md

```diff
@@ -0,0 +1,257 @@
+      allowed or denied respectively.
+
+    * The data extracted from the request are expected by your authorization policy.
+
+1. You should see something similar to the following:
+
+    {{< text plain >}}
+    ...
+    [2018-07-26 20:39:18.060][152][debug][rbac] external/envoy/source/extensions/filters/http/rbac/rbac_filter.cc:79] checking request: remoteAddress: 10.60.0.139:51158, localAddress: 10.60.0.93:9080, ssl: uriSanPeerCertificate: spiffe://cluster.local/ns/istio-system/sa/istio-ingressgateway-service-account, subjectPeerCertificate: O=, headers: ':authority', '35.238.0.62'
+    ':path', '/productpage'
+    ':method', 'GET'
+    'upgrade-insecure-requests', '1'
+    'user-agent', 'Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/67.0.3396.99 Safari/537.36'
+    'dnt', '1'
+    'accept', 'text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,image/apng,*/*;q=0.8'
+    'accept-encoding', 'gzip, deflate'
+    'accept-language', 'en-US,en;q=0.9,zh-CN;q=0.8,zh;q=0.7'
+    'x-forwarded-for', '10.60.0.1'
+    'x-forwarded-proto', 'http'
+    'x-request-id', 'e23ea62d-b25d-91be-857c-80a058d746d4'
+    'x-b3-traceid', '5983108bf6d05603'
+    'x-b3-spanid', '5983108bf6d05603'
+    'x-b3-sampled', '1'
+    'x-istio-attributes', 'CikKGGRlc3RpbmF0aW9uLnNlcnZpY2UubmFtZRINEgtwcm9kdWN0cGFnZQoqCh1kZXN0aW5hdGlvbi5zZXJ2aWNlLm5hbWVzcGFjZRIJEgdkZWZhdWx0Ck8KCnNvdXJjZS51aWQSQRI/a3ViZXJuZXRlczovL2lzdGlvLWluZ3Jlc3NnYXRld2F5LTc2NjY0Y2NmY2Ytd3hjcjQuaXN0aW8tc3lzdGVtCj4KE2Rlc3RpbmF0aW9uLnNlcnZpY2USJxIlcHJvZHVjdHBhZ2UuZGVmYXVsdC5zdmMuY2x1c3Rlci5sb2NhbApDChhkZXN0aW5hdGlvbi5zZXJ2aWNlLmhvc3QSJxIlcHJvZHVjdHBhZ2UuZGVmYXVsdC5zdmMuY2x1c3Rlci5sb2NhbApBChdkZXN0aW5hdGlvbi5zZXJ2aWNlLnVpZBImEiRpc3RpbzovL2RlZmF1bHQvc2VydmljZXMvcHJvZHVjdHBhZ2U='
+    'content-length', '0'
+    'x-envoy-internal', 'true'
+    'sec-istio-authn-payload', 'CkVjbHVzdGVyLmxvY2FsL25zL2lzdGlvLXN5c3RlbS9zYS9pc3Rpby1pbmdyZXNzZ2F0ZXdheS1zZXJ2aWNlLWFjY291bnQSRWNsdXN0ZXIubG9jYWwvbnMvaXN0aW8tc3lzdGVtL3NhL2lzdGlvLWluZ3Jlc3NnYXRld2F5LXNlcnZpY2UtYWNjb3VudA=='
+    , dynamicMetadata: filter_metadata {
+      key: "istio_authn"
+      value {
+        fields {
+          key: "request.auth.principal"
+          value {
+            string_value: "cluster.local/ns/istio-system/sa/istio-ingressgateway-service-account"
+          }
+        }
+        fields {
+          key: "source.principal"
+          value {
+            string_value: "cluster.local/ns/istio-system/sa/istio-ingressgateway-service-account"
+          }
+        }
+      }
+    }
+
+    [2018-07-26 20:39:18.060][152][debug][rbac] external/envoy/source/extensions/filters/http/rbac/rbac_filter.cc:88] shadow denied
+    [2018-07-26 20:39:18.060][152][debug][rbac] external/envoy/source/extensions/filters/http/rbac/rbac_filter.cc:98] enforced allowed
+    ...
+    {{< /text >}}
+
+    It means there is a `GET` request at path `/productpage` and is allowed by the policy.
```

Comment:
This output means there is a `GET` request at path `/productpage` and the policy allows the request.